tm-bench/main.go

```diff
@@ -100,6 +108,9 @@ Examples:
 		case l := <-blockLatencyCh:
 			stats.BlockLatency.Update(int64(l))
 		case <-ticker.C:
+			if outputFormat != "json" {
```

Comment:
are we just logging a number (without any info?)

Reply:
This is a leftover debug message. I'll clean it up, thanks for noting.